k8s.gcr.io/images/k8s-staging-etcd/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
I copied owners from https://github.com/kubernetes/kubernetes/blob/master/cluster/images/etcd/OWNERS
and added myself. Should I add myself to the original OWNERS file first?

Reply:
I would like to see you added to that file, yes, but I don't consider it a blocker for this